index.html

```diff
@@ -0,0 +1,50 @@
+
+
+<!DOCTYPE html>
+<html>
+   <head>
+     <title>Dr. Borlaug's life</title>
+     <link rel="stylesheet" href="./style.css">
+   </head>
+   <body>
+     <main id="main">
+       <h1 id="title">Dr. Norman Borlaug</h1>
+       <p>The man who saved a billion lives</p>
+       <figure id="img-div">
+    <img id="image" src="https://www.write-out-loud.com/images/commemorative-speech-candle.jpg" alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists">
+    <figcaption id="img-caption">
+    Choose the Right Synonym for tribute
+ENCOMIUM, EULOGY, PANEGYRIC, TRIBUTE, CITATION mean a formal expression of praise. ENCOMIUM implies enthusiasm and warmth in praising a person or a thing.  received encomiums from literary critics  EULOGY applies to a prepared speech or writing extolling the virtues and services of a person.  delivered the eulogy at the
+    </figcaption>
+  </figure>
+       
+     <section id="tribute-info">
+      <!-- <p>
+        "Borlaug's life and achievement are testimony to the far-reaching
+        contribution that one man's towering intellect, persistence and
+        scientific vision can make to human peace and progress."
+      </p> -->
```

Comment:
Your code contains unused code as comments. this is not recommended since unused code makes your code untidy. please remove commented unused code.